esp/src/eclwatch/WsWorkunits.js

```diff
@@ -141,11 +141,6 @@ define([
             return ESPRequest.send("WsWorkunits", "WUGetNumFileToCopy", params);
         },
 
-        CreateQuerySetStore: function (options) {
```

Comment:
In this case the linting highlighted that the content of this function was invalid and I searched and no-one is calling it.